pkg/ipcache/ipcache.go

```diff
@@ -511,7 +511,7 @@ func (ipc *IPCache) deleteLocked(ip string, source source.Source) (namedPortsCha
 
 	cachedIdentity, found := ipc.ipToIdentityCache[ip]
 	if !found {
-		scopedLog.Debug("Attempt to remove non-existing IP from ipcache layer")
+		scopedLog.Error("Attempt to remove non-existing IP from ipcache layer")
```

Comment:
I'm a little concerned that this will cause noisy logs in some scenario, but I'm OK to try this out and see what reports come through up until 1.13 final.

Reply:
I have the same concern, but we fail on any error in CI, so hopefully we should catch at least the most obvious scenarios.

Reply:
Well, CI is currently failing because of that error log. See, it's working :sweat_smile: